src/applications/personalization/profile/components/personal-information/fields/VAPPhoneField.jsx

```diff
@@ -11,7 +11,7 @@ import PhoneNumberWidget from 'platform/forms-system/src/js/widgets/PhoneNumberW
 import VAPProfileField from 'vet360/containers/VAPProfileField';
 import ReceiveTextMessages from 'vet360/containers/ReceiveTextMessages';
 
-import PhoneEditView from './PhoneEditView';
+import PhoneEditView from 'platform/user/profile/vet360/components/PhoneField/PhoneEditView';
```

Comment:
Can/should the `PhoneEditView` this is importing also be moved out of `platform`?

Reply:
Yes, I have a follow-up PR that does that also 😄